vmdb/app/models/ems_event/parsers/openstack_infra.rb

```diff
@@ -17,9 +17,9 @@ def self.event_to_hash(event, ems_id)
     }
 
     payload = event[:content]["payload"]
-    event_hash[:host_ems_ref] = Host.where('ems_ref_obj LIKE ?', "%#{payload['instance_id']}%").first.try(:ems_ref) if payload.has_key? "instance_id"
+    event_hash[:host_ems_ref]              = payload["node"]              if payload.has_key? "node"
```

Comment:
please use `key?`  here and below.  I'm ok with adding a commit to fix all of them in this method to use `key?` so your changes are isolated.